packages/manager/modules/pci/src/projects/project/storages/object-storage/users/delete/delete.routing.js

```diff
@@ -1,32 +1,28 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state(
-    'pci.projects.project.storages.containers.users.delete',
-    {
-      url: '/delete?userId',
-      views: {
-        modal: {
-          component: 'pciProjectStoragesContainersUsersDeleteComponent',
-        },
-      },
-      layout: 'modal',
-      params: {
-        userId: null,
-      },
-      resolve: {
-        userId: /* @ngInject */ ($stateParams) => $stateParams.userId,
-        user: /* @ngInject */ (
-          projectId,
-          userId,
-          PciStoragesObjectStorageService,
-        ) => PciStoragesObjectStorageService.getUserDetails(projectId, userId),
-        credentials: /* @ngInject */ (
-          userId,
-          projectId,
-          PciStoragesObjectStorageService,
-        ) =>
-          PciStoragesObjectStorageService.getS3Credentials(projectId, userId),
-        breadcrumb: () => null,
+  $stateProvider.state('pci.projects.project.storages.objects.users.delete', {
+    url: '/delete?userId',
```

Comment:
```suggestion
    url: '/:userId/delete',
```

Reply:
Ill take this up in the main branch...as this was showing up because of conflicts here